contracts/HATVaults.sol

```diff
@@ -262,47 +263,47 @@ contract  HATVaults is Governable, ReentrancyGuard {
                 uint256 _bountyVestingDuration,
                 uint256 _bountyVestingPeriods);
 
-    event SetPool(uint256 indexed _pid, uint256 indexed _allocPoint, bool indexed _registered, string _descriptionHash);
+    event SetPool(uint256 indexed _pid, uint256 indexed _allocPoint, bool indexed _registered, bool _depositPause, string _descriptionHash);
     event Claim(address indexed _claimer, string _descriptionHash);
     event SetBountySplit(uint256 indexed _pid, BountySplit _bountySplit);
     event SetBountyLevels(uint256 indexed _pid, uint256[] _bountyLevels);
     event SetFeeSetter(address indexed _newFeeSetter);
-    event SetPoolFee(uint256 indexed _pid, uint256 _newFee);
+    event SetPoolWithdrawalFee(uint256 indexed _pid, uint256 _newFee);
     event SetPendingBountyLevels(uint256 indexed _pid, uint256[] _bountyLevels, uint256 _timeStamp);
 
     event SwapAndSend(uint256 indexed _pid,
                     address indexed _beneficiary,
-                    uint256 indexed _amountSwaped,
+                    uint256 indexed _amountSwapped,
                     uint256 _amountReceived,
                     address _tokenLock);
 
-    event SwapAndBurn(uint256 indexed _pid, uint256 indexed _amountSwaped, uint256 indexed _amountBurned);
+    event SwapAndBurn(uint256 indexed _pid, uint256 indexed _amountSwapped, uint256 indexed _amountBurned);
     event SetVestingParams(uint256 indexed _pid, uint256 indexed _duration, uint256 indexed _periods);
     event SetHatVestingParams(uint256 indexed _duration, uint256 indexed _periods);
 
-    event ClaimApproved(address indexed _committee,
```

Comment:
no wait, do not remove the indexed addresses, even if it is msg.sender (indexed means: you can search for them in the events log, which is useful)